course7/homeworks/patrickkostjens/ls/ls.cc

```diff
@@ -0,0 +1,13 @@
+#include <iostream>
+#include <boost/filesystem.hpp>
+
+int main(int argc, char** argv)
+{
+	boost::filesystem::path current_directory(argc >= 2 ? argv[1] : ".");
+
+	boost::filesystem::directory_iterator end;
+	for (boost::filesystem::directory_iterator itr(current_directory); itr != end; ++itr) 
+	{
+		std::cout << itr->path().filename().string() << std::endl;
```

Comment:
is the call to ```string()``` necessary?

Reply:
Not strictly necessary, but if it's not added, all the elements will be printed with double quotes around them.